pkgs/development/python-modules/dmgbuild/default.nix

```
@@ -0,0 +1,55 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  ds-store,
+  importlib-resources,
+  mac-alias,
+}:
+
+buildPythonPackage rec {
+  pname = "dmgbuild";
+  version = "1.6.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "dmgbuild";
+    repo = "dmgbuild";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-PozYxmXumFnptIgb4FM4b/Q5tx0MIS2bVw2kCuGucA8=";
+  };
+
+  postPatch = ''
+    # relax all deps
+    substituteInPlace pyproject.toml \
+      --replace-fail "==" ">="
+  '';
```

```suggestion
  pythonRelaxDeps = true;
```
You don't need the hook as it's python builder.

See https://nixos.org/manual/nixpkgs/unstable/#using-pythonrelaxdepshook

---

`pythonRelaxDeps` is not work in this package.

https://github.com/dmgbuild/dmgbuild/blob/198eb9957ebb5b12e8524bb2d4dd63bc047c6594/pyproject.toml#L3